src/earn/EarnConfirmationScreen.test.tsx

```diff
@@ -103,48 +103,52 @@ describe('EarnCollectScreen', () => {
     store.clearActions()
   })
 
-  it('renders total balance, rewards, apy and gas after fetching rewards and preparing tx', async () => {
+  it('renders total balance, rewards and gas after fetching rewards and preparing tx', async () => {
```

Comment:
Can you add a unit tests with inputAmount to verify the ui's and prepareWithdrawAndClaimTransactions params change as expected?

Reply:
Added in 3958cef0fb948d09e9617883383d7d86b6e8e805!